Frameworks/UIKit/UIView.mm

```diff
@@ -768,29 +768,20 @@ - (instancetype)initWithCoder:(NSCoder*)coder {
             ![coder decodeInt32ForKey:@"UIViewDoesNotTranslateAutoresizingMaskIntoConstraints"];
     }
 
-    if ([coder containsValueForKey:@"UIViewContentCompressionResistancePriority"]) {
-        id sizeObj = [coder decodeObjectForKey:@"UIViewContentCompressionResistancePriority"];
-        if ([sizeObj isKindOfClass:[NSString class]]) {
-            const char* stretchStr = [sizeObj UTF8String];
-            sscanf_s(stretchStr,
-                     "{%f, %f}",
-                     &priv->_contentCompressionResistancePriority.width,
-                     &priv->_contentCompressionResistancePriority.height);
-        } else {
-            CGSize* size = (CGSize*)[sizeObj bytes];
-            memcpy(&priv->_contentCompressionResistancePriority, size, sizeof(CGRect));
-        }
+    if ([coder containsValueForKey:@"UIViewHorizontalHuggingPriority"]) {
+        priv->_contentHuggingPriority.width = [coder decodeFloatForKey:@"UIViewHorizontalHuggingPriority"];
     }
 
-    if ([coder containsValueForKey:@"UIViewContentHuggingPriority"]) {
```

Comment:
I don't know where this came from, but we weren't writing it out to the nib in Xib2Nib.  I almost left it in with a TODO, but felt that it was just cleaner to remove it.  

Also, passing these distinct priority values across as a CGSize is a bit...weird, so I'd be surprised if that's how it's done on the reference platform.

---
In reply to: [107229813](https://github.com/Microsoft/WinObjC/pull/2269#discussion_r107229813) [](ancestors = 107229813)